CMakeLists.txt

```diff
@@ -15,6 +15,13 @@ project(Glu CXX)
 include(UnixCompileRules)
 include(FetchLLVM)
 
+option(ENABLE_ASAN "Enable AddressSanitizer" OFF)
+
+if(ENABLE_ASAN)
+    add_compile_options(-fsanitize=address -fno-omit-frame-pointer)
+    add_link_options(-fsanitize=address)
+endif()
```

Comment:
Maybe this option should be scoped in a specific targets 🤔 @LindonAliu @Snowy1803

Reply:
For me it's not necessary and I think it's better this way for debugging